code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -378,6 +378,15 @@
 	max_w_class = WEIGHT_CLASS_SMALL
 	max_combined_w_class = 18
 	resistance_flags = FIRE_PROOF
+	use_item_overlays = TRUE // So it will still show tools in it in case sec get lazy and just glance at it.
```

Comment:
This comment applies less to the normal military belt, can you change this comment?